FrontEnd/package.json

```diff
@@ -9,14 +9,19 @@
     "lint": "eslint src --ext js,jsx --report-unused-disable-directives --max-warnings 0",
     "preview": "vite preview"
   },
+  "license": "MIT",
   "dependencies": {
     "@chakra-ui/react": "^2.6.0",
-    "@emotion/react": "^11.10.8",
-    "@emotion/styled": "^11.10.8",
+    "@emotion/react": "^11.11.0",
+    "@emotion/styled": "^11.11.0",
+    "@mui/icons-material": "^5.11.16",
+    "@mui/material": "^5.12.3",
     "framer-motion": "^10.12.6",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
-    "react-icons": "^4.8.0"
+    "react-icons": "^4.8.0",
+    "react-type-animation": "^3.0.1",
+    "react-typing-animation": "^1.6.2"
```

Comment:
Are both of these libraries required in the project?

Reply:
No, only the first one, I'll remove the second library!